### ISSUE TITLE
The UI doesn't appear at all

### ISSUE THREAD

**Issue by denizonm** [jeffvli/sonixd#278]
The tray icon is present and works, but the main ui doesn't. 

I wondered if this could have been a multiple monitor issue but it doesn't seem to be. I'm not sure but i think this happened when i updated to the newest version.

The window appears in the taskbar and upon hovering, everything looks normal. Pressing the window makes it active.

Love the program!

**Comment by denizonm** [comment]
I'm on windows 10, a clean install hasn't solved the problem.

**Comment by jeffvli** [comment]
Can you try making the application active by clicking on it in the taskbar and the using the `Windows Key` + `Arrow Key` to try moving it? Maybe it got stuck outside of your monitor zones.

You can also try resetting your settings by deleting the `settings.json` file in `%appdata%/sonixd%` and then restarting the app.

**Comment by slsnow** [comment]
I had this same issue, and for some reason deleting the settings.json file did fix it. Thanks.

**Comment by jeffvli** [comment]
If this happens in the future for anyone else, please provide me your broken `settings.json` file so I can debug which setting is being corrupted. Though, **make sure you sanitize the json file by removing settings like your token, password, username, etc. before posting here**.

**Comment by bwat47** [comment]
broken settings file attached
[settings.zip](https://github.com/jeffvli/sonixd/files/8546802/settings.zip)

**Comment by jeffvli** [comment]
@bwat47 Thanks, seems to actually just be the persisted window size/position breaking.

Modify those values to something like 0, 0 for x, y and 1280, 720 for width and height and I believe it should resolve it. I'll fix this for the next version.
```json
  "windowPosition": {
    "x": -25600,
    "y": -25600,
    "width": 0,
    "height": 0
  }
```

**Comment by owospace** [comment]
Just ran into this issue; as mentioned above, window size is brokey, either change the width/height/location (see below if you'd like) or delete settings.json in %appdata%/sonixd to fix ^^

Broken configuration reports:
```
  "windowPosition": {
    "x": -32000,
    "y": -32000,
    "width": 0,
    "height": 0
  }
```

'Unbrokey configuration':
```
  "windowPosition": {
    "x": 104,
    "y": 50,
    "width": 1024,
    "height": 728
  }
```

**Comment by jeffvli** [comment]
I'll push a fix soon to remove this behavior since it seems to be breaking more often than it's working.

**Comment by 132ikl** [comment]
When do you think you'll be able to make a release with the fix? This happens almost every day for me 😢

**Comment by jeffvli** [comment]
@132ikl I can fix this in the next release (somewhat slipped my mind). By any chance, do you have minimize to tray or exit to tray enabled? That's one thing that I'm thinking may be triggering this issue.

**Comment by bwat47** [comment]
@jeffvli in my case I do not have either of these options enabled

the issue seems to occur randomly (I've seen it happen on every machine that I use sonixd on), and I haven't been able to find any specific steps to re-create

**Comment by jeffvli** [comment]
Fixed in 0.15.3
